Makefile

```diff
@@ -16,16 +16,30 @@ start:
 	docker network inspect iloop || docker network create iloop
 	docker-compose up -d --build
 
+## Run all QA targets
+qa: test flake8 isort license
```

Comment:
Do you think it makes sense to offload these into different tox environments in future?

Reply:
Yes, maybe that would make sense. Although from my understanding, what tox does best is combining multiple environments, which isn't that relevant for an application package where we typcially just target a single, recent environment.

Regardless, collecting all QA stuff under tox and not duplicating it in CI and Make config would be nice.